data/Templates/Stu3ToR4/DataType/DataRequirement/_CodeFiter.liquid

```diff
@@ -0,0 +1,19 @@
+{% mergeDiff msg -%}
```

Comment:
`fiter` or `filter`?

Reply:
Just curious how is `DataRequirement` defined? Why are `DataRequirement` and `Dosage` saved in separate subfolders?

Reply:
Thanks, should be `codeFilter`.
`DataRequirement` and `Dosage` are [MetaData Types](https://www.hl7.org/fhir/metadatatypes.html#DataRequirement) in FHIR rather than resources. So we put them in DataType folder.